core/profiles/standard/standard.install

```diff
@@ -248,7 +248,7 @@ function standard_install() {
 
   // Create an image field named "Image", enabled for the 'post' content type.
   // Many of the following values will be defaulted, they're included here as an illustrative examples.
-  // See https://api.backdropcms.org/api/function/field_create_field/1
+  // See https://docs.backdropcms.org/api/backdrop/core%21modules%21field%21field.crud.inc/function/field_create_field/1
```

Comment:
This looks funny to me, but the link is correct. I think we have a problem with the API module that's generating this link. I have opened https://github.com/backdrop-contrib/api/issues/7 to investigate further, but that should not block this issue from proceeding.